doc/ref_model/chapters/chapter04.md

```diff
@@ -278,47 +278,55 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
-| Management Interface                  | Specifies the bandwidth of management interface/s                                                                                                                                                       |
-
+| Root Disk                  | Specifies the size of '/' root partition. It can be considered an Ephemeral Disk |
+| CPU Parameters                  | Capability to associate virtual CPUs to the physical CPUs, and enable specific CPU architectures |
+| CPU Topology                  | NUMA topology and CPU pinning features can minimize latency and improve performance. Check which architecture is supported by the compute nodes.  |
+| Network Parameters                  | Enable network capabilities allowing improvements on network performance (make sure that current deployment allows it, otherwise, it can lead either to a decrease on performance or instability) |
 <p align="center"><b>Table 4-12:</b> Compute Flavour Geometry Specification.</p>
 
 <a name="4.2.1.1"></a>
-#### 4.2.1.1 Predefined Compute Flavours
-The intent of the following Flavours list is to be comprehensive and yet effective to cover both IT and NFV workloads. The compute Flavours are specified relative to the “large” Flavour. The “large” Flavour configuration consists of 4 vCPUs, 8 GB of RAM and 80 GB of local disk, and the resulting virtual compute instance will have a management interface of 1 Gbps. The “medium” Flavour is half the size of a large and small is half the size of medium. The tiny Flavour is a special sized Flavour.
+#### 4.2.1.1 Predefined Workload Profiles
+
+The following tables provide a descriptive approach on how a profile can be created to suit the needs of a workload, so it can be onboarded. This will minimize several challenges related to Infrastructure management:
+
+* Workloads can be deployed according to specific profiles (Basic; High Performance), and can be configured with a lower level of granularity by using profile extensions
+* Profile extensions allow a more precise configuration of a workload (e.g. GPU, high, speed network, Edge deployment)
+* Cloud infrastructure "scattering" is minimized
+* Workload optimisation by using existing profiles and its extensions
+* Better usage of Cloud Objects (Memory;Processor;Network;Storage)
+
+
+The intent of the following profiles is to match the cloud infrastructure most common needs, and allow a more  comprehensive configuration in case needed. <br>
+
+NOTE: In Kubernetes based environments these are the resource requests of the containers in the pods. To get guaranteed resources the resource requests should be set to the same values as the resource limits, to get burstable resources the resource limits should be higher than the resource requests while to get best effort resources none of resource requests of resource limits should be set.
+There are two workload profiles available which can be selected:
```

Comment:
Need an extra line. Otherwise this sentence is part of the NOTE.

Reply:
```suggestion

There are two workload profiles available which can be selected:
```